flink-connector-mongodb-cdc/src/main/java/com/ververica/cdc/connectors/mongodb/source/config/MongoDBSourceOptions.java

```diff
@@ -90,6 +90,13 @@ public class MongoDBSourceOptions {
                     .defaultValue(1024)
                     .withDescription("The cursor batch size. Defaults to 1024.");
 
+    public static final ConfigOption<Integer> SNAPSHOT_BATCH_SIZE =
```

Comment:
This feature should split into another pr .Btw, document should be updated too. cc @Jiabao-Sun

Reply:
Temporarily replaced with batch size. I'll create pr, when the current one will be merged.

Reply:
> Temporarily replaced with batch size. I'll create pr, when the current one will be merged.

@tigrulya-exe MongoDB snapshot phase use cusor fetch , `batchSize` seems same to cusor fetch.

Reply:
Do you mean not to introduce a new configuration parameter for the batch size at the snapshot stage and keep it as it is now (use batchSize of streaming for shapshot queue size)? Ok